<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoldenAura</class>
 <widget class="QMainWindow" name="GoldenAura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1695</width>
    <height>959</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mistral</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 127);
</string>
    </property>
    <property name="text">
     <string>GOLDENAURA BEAUTY HOUSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butonKayitEkle">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>70</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>KAYIT EKLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butonKayitAra">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>270</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>KAYIT ARA</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>761</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mistral</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>MÜŞTERİ BİLGİLERİ</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>321</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ad                      :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lneAd">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Soyad                :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lneSoyad">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TC Kimlik No     :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lneTCK">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="inputMask">
         <string>99999999999</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefon No        :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lneTel">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="inputMask">
         <string>(999) 999 99 99</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCalendarWidget" name="cwDogum">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>392</width>
       <height>236</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>10</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Doğum Tarihi   :</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>90</y>
      <width>431</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mistral</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>                         İŞLEMLERİMİZ</string>
    </property>
    <widget class="QLabel" name="label_54">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mistral</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CİLT BAKIMI              :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mistral</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TIRNAK BAKIMI          :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbCiltBakimi">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>188</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Profosyonel Cilt Bakımı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leke Tedavisi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Akne Tedavisi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Anti-Aging Bakım</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dolgu</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Botoks</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbTirnakBakimi">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>188</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Jel Tırnak</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kalıcı Oje</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manikür</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pedikür</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Manikür+Pedikür</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_57">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mistral</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>KALICI MAKYAJ          :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbKaliciMakyaj">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>188</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Mikroblaiding</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>İpek Kirpik</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lifting</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kalıcı Makyaj</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kalıcı Ruj</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_56">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Mistral</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>AĞDA-EPİLASYON        :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbEpilasyon">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>188</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Lazer Epilasyon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Buz Epilasyon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kaş-Bıyık</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sir Ağda</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="butonListele">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>170</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>KAYIT LİSTELE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butonKayitDuzenle">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>360</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>KAYIT DÜZENLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="butonKayitSil">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>460</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>KAYIT SİL</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>761</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mistral</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>YETKİLİ GÜZELLİK UZMANI</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ad-Soyad                   :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbGuzellikUzmani">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>188</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Asel Naz Kılıç</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Deniz Arden</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ayşe Selen Kuşakçı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mine Beyatlı</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Asu Pelin Yağmacı</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>470</y>
      <width>331</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mistral</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>ÖDEME BİLGİLERİ</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lneIslemTutari">
       <property name="font">
        <font>
         <family>Calibri</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="inputMask">
        <string>999999999999 TL</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ödenen Tutar                       :    </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lneOdenen">
       <property name="font">
        <font>
         <family>Calibri</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="inputMask">
        <string>999999999999 TL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kalan Tutar                            :   </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lneKalan">
       <property name="font">
        <font>
         <family>Calibri</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="inputMask">
        <string>999999999999 TL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>İşlem Tutarı                           :    </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="Liste">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>1671</width>
      <height>192</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>4</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>5</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>6</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>7</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>8</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>11</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>12</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>13</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>14</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="butonCikis">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>560</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>ÇIKIŞ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblKayitSayisi">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>KAYIT SAYISI  :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1695</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuHakkinda">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHakkinda"/>
   </widget>
   <addaction name="MenuHakkinda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionHakkinda">
   <property name="text">
    <string>Hakkinda</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
